.travis.yml

```diff
@@ -1,31 +1,33 @@
 language: python
 sudo: false
+cache: pip
+git:
+   depth: 1
 python:
 - 2.6
 - 2.7
-- 3.2
```

Comment:
It does, but some of your test dependencies require 3.3 or above. I ran out of steam at the end of this, so just removed them.

It may be worth re-adding if only for the pypy3 support, but perhaps as a separate changeset?